pandas/tests/indexes/test_multi.py

```diff
@@ -1457,7 +1486,16 @@ def test_equals_multi(self):
 
         index = MultiIndex(levels=[major_axis, minor_axis],
                            labels=[major_labels, minor_labels])
-        self.assertFalse(self.index.equals(index))
+        assert not self.index.equals(index)
+
+    def test_equals_missing_values(self):
+        # make sure take is not using -1
+        i = pd.MultiIndex.from_tuples([(0, pd.NaT),
+                                       (0, pd.Timestamp('20130101'))])
+        result = i[0:1].equals(i[0])
```

Comment:
was to make things a bit easier on collsision check, but you are right, not necessary (or correct), reverted.